lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -342,6 +342,28 @@ abstract class Image {
   String toString() => '[$width\u00D7$height]';
 }
 
+// no-op in web target
```

Comment:
@yjbanov how hard would it be to calculate this timing info for the web target?

Reply:
Web target rely on chromium to render Web Image. AFAIK there is no W3C interface to fetch perf-infos on the per-file level.

Reply:
It is fine to leave this unimplemented on the web. it could be good to file a bug to track just in case at some point this makes sense to implement on the web.